openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -2122,7 +2122,7 @@ boost::optional<IdfObject> ForwardTranslator::translateAndMapModelObject(ModelOb
       model::ShadingControl shadingControl = modelObject.cast<ShadingControl>();
       retVal = translateShadingControl(shadingControl);
       break;
-    }    
```

Comment:
@nmerket in the future you might set your editor to ignore trailing whitespace, just makes the diffs harder to compare